<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>613</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNovoCadastro">
       <property name="minimumSize">
        <size>
         <width>611</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Novo Cadastro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAtualizarDados">
       <property name="minimumSize">
        <size>
         <width>611</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Atualizar Dados</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNovoPagamento">
       <property name="minimumSize">
        <size>
         <width>611</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Novo Pagamento</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMostrarSaldo">
       <property name="minimumSize">
        <size>
         <width>611</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Mostrar Saldo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHistorico">
       <property name="minimumSize">
        <size>
         <width>611</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Hist√≥rico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAcerto">
       <property name="minimumSize">
        <size>
         <width>611</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Acerto Gastos</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>811</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{background-color: qlineargradient(spread:pad, x1:0.512, y1:0, x2:0.518, y2:1, stop:0 rgba(254, 255, 172, 255), stop:1 rgba(255, 170, 127, 255));}
</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
